### PR TITLE
add - rules page

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Add rules page

Preview - https://deploy-preview-2--vishnu-quiz.netlify.app/pages/rules.html

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
Looks good, have better indentation. Maybe change your code formatter or change its settings.

**Comment by Vishnu-Aithal** [comment]
> Looks good, have better indentation. Maybe change your code formatter or change its settings.

Thanks For the Review Will Do it

File: pages/rules.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  <link rel="stylesheet" href="https://some-ui-components.netlify.app/export.css" />
+  <link rel="stylesheet" href="/stylesheets/rules.css" />
+  <title>Quiz-Rules</title>
+</head>
+
+<body>
+  <nav class="nav-bar bg-secondary-light shadow-sm">
+    <div class="nav-bar__header heading-md text-bold clr-white me-2">
+      <a class="nav-bar__link" href="#">Quizz</a>
```
Comment (vrinda-mahajan) [inline pages/rules.html:17]: I think you forgot to add the home page link here.

File: stylesheets/rules.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.rules {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
```
Comment (vrinda-mahajan) [inline stylesheets/rules.css:5]: Suggestion- You can make helper classes for flex.

**Review by vrinda-mahajan — APPROVED** [review]
Everything else looks good.

File: pages/rules.html (hunk 1 above)
Comment (Vishnu-Aithal) [reply pages/rules.html:17]: yep , will add it.
Thanks For reviewing

**Pull request merged**